src/include/apparmor/aa-config.rb

```diff
@@ -33,20 +33,26 @@ def initialize_apparmor_aa_config(include_target)
       Yast.import "Label"
       Yast.import "Popup"
       Yast.import "Wizard"
+      Yast.import "Service"
     end
 
     def changeAppArmorState(aaEnabled)
       error = nil
       aaAction = ""
+      service = Yast::ServiceClass.new
```

Comment:
this is not needed, service already contain instance `Yast::Service` on which you can do calls like `Yast::Service.enable("apparmor")`